HTML/CSS/JavaScript Files

KareezAy opened the pull request:
#### What does this PR do?
* It is a pike to explore how a user can register, login, create a business as well as post a review for a business for all other users to view on a business profile
* This fixes how to handle updating and deleting a business.
* This simplifies the use of buttons by creating multiple buttons styled by CSS.

#### Where should the reviewer start?
The changes can be viewed on an active site: kareezay.github.io

#### Any background context you want to provide?
The commits consist of
1. index.html 
2. dashboard.html
3. f-dashboard.html

Commit:
Created HTML/CSS templates

The HTML and CSS templates can be found in the WeConnect_project/designs/UI directory

[finishes #155553023]
[finishes #155554022]
[starts #155543228]

Commit:
Updated HTML/CSS/JS files

I added content on the templates created for a better design

[Finishes #155559443]
[Finishes #155558372]
[Finishes #155560709]
[Finishes #155560627]

Commit:
HTML pages

User can be able to create a business
Created a business catalog

[Finishes #155558642]
[Finishes #155558887]
[Finishes #155559268]
[Starts #155559127]

Commit:
Add business reviews and rating

Remove old bootstrap tags

[Finishes #155585301]
[Finishes #155492137]

jmnyarega commented on designs/UI/businesses.html:
```diff
@@ -0,0 +1,106 @@
+    
+    </body>
+    
+    </html>
```
It's highly advisable to a **one** extra line at the end of your file

KareezAy replied:
Understood

jmnyarega commented on designs/UI/create-businesses.html:
```diff
@@ -0,0 +1,120 @@
+
+                </div>
+            </div>
+            <div class="row" style="padding-top: 60px; clear: both;"></div>
```
Use external style instead of inline

KareezAy replied:
I will do the cleanup

jmnyarega commented on designs/UI/dashboard.html:
```diff
@@ -0,0 +1,131 @@
+                </div>
+                </div>
+
+    </body>
```
Please learn how to format your code, it is much cleaner when formatted

Commit:
Update README.md
Format code
Add extra line on all templates

[Delivers #155602859]
[Delivers #155606289]
[Delivers #155606405]
[Delivers #155606504]

Commit:
Add a new button style

Commit:
Create delete and update business buttons

Commit:
Add confirmation page to delete business

Commit:
Add business update page

[Delivers #155559127]
[Delivers #155492071]

Commit:
Post review on business profile

[Delivers #155492115]

Commit:
Add a server JS file for testing user authorization

Commit:
Reposition token placeholder

Commit:
Add name of app on Header
